README.md

```diff
@@ -102,9 +101,25 @@ See [BUILD.md](BUILD.md)
 
 See http://www.konstructs.org/download/
 
+### Connecting to a Server
+
+There are multiple ways to connect to a server. The most common way is to just start the game and follow the on-screen prompts.
+
+A more advanced way is through commandline arguments. The advantage of this is, that commandline arguments can be included in link files (on Windows .lnk, on Linux sometimes calls "Starter") to automatically connect to your favorite server!
```

Comment:
Ah, that's probably just the translation then ... gnome/unity usually call these "Desktop shortcut", "Launcher shortcut", and so on ... they are usually `.desktop`-files. http://standards.freedesktop.org/desktop-entry-spec/latest/